src/main/resources/add-ons-to-index.json

```
@@ -1558,6 +1558,39 @@
         "package": "facelist"
       }
     },
+    {
+      "uid":"org.openmrs.module.spa",
+      "type":"OMOD",
+      "name":"Single Page Application",
+      "maintainers":[
+        {
+          "name":"Daniel Kayiwa"
+        },
+        {
+          "name":"Burke Mamlin"
+        },
+        {
+          "name":"Brandon Istenes"
+        }
+      ],
+      "links":[
+        {
+          "rel":"documentation",
+          "href":"https://wiki.openmrs.org/display/projects/Frontend+Implementer+Documentation",
+          "title":"OpenMRS Wiki"
+        },
+        {
+          "rel":  "source",
+          "href": "https://github.com/openmrs/openmrs-module-spa",
```

```suggestion
          "href": "https://github.com/openmrs/openmrs-module-spa"
```

JSON is very strict. You cannot have a comma on the end of a record unless the next thing is a new value.

---

Thanks @ibacher , sure i have resolved that